ecosystem/typescript/sdk/package.json

```diff
@@ -5,6 +5,14 @@
   "engines": {
     "node": ">=11.0.0"
   },
+  "sideEffects": false,
+  "type": "module",
```

Comment:
This would default the module to ESM. Last time I found some issues with `./dist/aptos.esm.js`. Have you tested if `./dist/aptos.esm.js` works in your App? Let me know. 
Also, we can create a dual module without hard-coding `"type": "module"`.  It would be important for an environment that has to load CJS modules.